sci-libs/pdal/files/pdal-2.4.3-fix-gdal37.patch

```diff
@@ -0,0 +1,21 @@
+diff -Nru PDAL-2.5.3-src-orig/pdal/Geometry.cpp PDAL-2.5.3-src/pdal/Geometry.cpp
```

Comment:
Please include commit links for patches.

Reply:
@espinafre can you add this information?

Reply:
and take into account upstream may soon change their mind https://github.com/PDAL/PDAL/pull/4043

Reply:
@thesamesam links are in now :+1: